src/nvim/window.c

```diff
@@ -907,8 +907,10 @@ void ui_ext_win_position(win_T *wp)
       int comp_col = (int)col - (east ? wp->w_width_outer : 0);
       comp_row += grid->comp_row;
       comp_col += grid->comp_col;
-      comp_row = MAX(MIN(comp_row, Rows - wp->w_height_outer - 1), 0);
-      comp_col = MAX(MIN(comp_col, Columns - wp->w_width_outer), 0);
+      comp_row = MAX(MIN(comp_row,
+                         Rows - wp->w_height_outer - (p_ch > 0 ? 1 : 0)), 0);
+      comp_col = MAX(MIN(comp_col,
+                         Columns - wp->w_width_outer), 0);
```

Comment:
```suggestion
      comp_row = MAX(MIN(comp_row, Rows - wp->w_height_outer - (p_ch > 0 ? 1 : 0)), 0);
      comp_col = MAX(MIN(comp_col, Columns - wp->w_width_outer), 0);
```

Use the 100 column line width properly